src/components/Card/README.md

```diff
@@ -436,6 +436,89 @@ Use to be able to use custom React elements as header content.
 </Card>
 ```
 
+### Full example of card
```

Comment:
Looks great! I wonder if this is the best name? @sadiesaurus and @selenehinkley do you have thoughts on what to name the examples in the style guide that include all of the above examples?

Full example?
Complete example?
Card with all elements?

Reply:
Good question! I did some browsing through other component examples, and there's definitely no established convention right now. That said, I did encounter `Top bar with all of its elements`:

![alt](https://screenshot.click/06-46-jhmse-ulmyl.jpg) 

So I might lean in that direction. It also aligns well with the other "Card" example titles.